lib/rules/no-test-import-export.js

```diff
@@ -58,3 +61,16 @@ module.exports = {
     };
   },
 };
+
+function isTestHelperImportSource(importSource) {
+  return (
+    importSource.startsWith('tests/helpers/') ||
+    importSource.includes('/tests/helpers/') ||
+    importSource.endsWith('/tests/helpers') ||
+    importSource === 'tests/helpers'
```

Comment:
Not sure. I suppose this line would handle either a file named `tests/helpers` or a file named `tests/helpers/index.js`.

Reply:
> Is it valid for tests/helpers to be a single file instead of a directory?

Yes, it is definitely possible (and allowed) to use a single `tests/helpers.js` file _or_ a `tests/helpers/index.js` file.